LiskHQ/lisk
# Create Round entity with CRUD operations and unit tests - Closes #2618

### What was the problem?

Round entity didn't exist

### How did I fix it?

By creating the Round entity

### How to test it?

- Build should pass
- run `mocha test/unit/storage/entities/round.js`

### Review checklist

* The PR resolves #2618
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
